cli/command/image/tree_test.go

```diff
@@ -115,6 +134,11 @@ func TestPrintImageTreeAnsiTty(t *testing.T) {
 			cli.In().SetIsTerminal(tc.stdinTty)
 			cli.Out().SetIsTerminal(tc.stdoutTty)
 			cli.Err().SetIsTerminal(tc.stderrTty)
+			if tc.noColorEnv {
+				t.Setenv("NO_COLOR", "1")
```

Comment:
(can be a follow-up), we should probably add `NO_COLOR` to the list of env-vars; https://github.com/docker/cli/blob/c44e8a072770cd5692158dc27b12e51d94898c1d/docs/reference/commandline/docker.md#environment-variables